src/PrestaShopBundle/Form/Admin/Sell/Product/Image/ImageDropzoneType.php

```diff
@@ -44,6 +44,8 @@
  */
 class ImageDropzoneType extends TranslatorAwareType
 {
+    use PrestaShopTranslatorTrait;
```

Comment:
Where is the `use` ?

Reply:
I know I removed it, but I didn't push yet